examples/e2ebot/config.yml

```diff
@@ -10,7 +10,8 @@ pipeline:
     min_ngram: 1
     max_ngram: 4
   - name: DIETClassifier
-    epochs: 200
+    epochs: 10

```

Comment:
let's keep `200`. Otherwise, our losses doesn't have a chance to train

Reply:
thanks - can confirm that with this additional NLU example, still works if I turn off memo and increase epochs back to 200